Libs/MRML/DisplayableManager/vtkMRMLViewInteractorStyle.cxx

```diff
@@ -476,7 +476,7 @@ void vtkMRMLViewInteractorStyle::SetInteractor(vtkRenderWindowInteractor *intera
   if (this->Interactor)
     {
     this->Interactor->AddObserver(vtkCommand::LeftButtonDoubleClickEvent, this->EventCallbackCommand, this->Priority);
-    this->Interactor->AddObserver(vtkCommand::MiddleButtonDoubleClickEvent, this->EventCallbackCommand, this->Priority);
+    //this->Interactor->AddObserver(vtkCommand::MiddleButtonDoubleClickEvent, this->EventCallbackCommand, this->Priority);
```

Comment:
Port commit https://github.com/Slicer/VTK/commit/99797966b61cb208a4c50d7cace0f875dbae69b9 to VTK upstream VTK. Worth noting that part of the functionality has already been integrated in kitware/VTK@f3ede3d11475b81

@Punzo  Is it something you could help with ?

Reply:
Ciao Jc, I loooked at the difference between https://github.com/Slicer/VTK/commit/99797966b61cb208a4c50d7cace0f875dbae69b9 and current VTK master and we need:

1) add _vtk_add_event(MiddleButtonDoubleClickEvent)\ in the vtkCommand.h  
2) apply
![image](https://user-images.githubusercontent.com/7985338/91655454-df511c80-eab0-11ea-81d2-96ef1e3cb70b.png)
to QVTKInteractorAdapter.

Should I open a PR in VTK (current VTK master? I think in general would be good to have that middle click signals, not just for Slicer)? or should we apply this only to the Slicer VTK repo?

---------------------------------------------------------------------------------------------------------------------------------------------------------

All the other stuff regarding the selectionButton in the vtkAbstractWidget can be discarded. They were used in the first draft version of the markups, but then we moved everything to MRML (and I forgot to clean the VTK branch, sorry!). @lassoan can you confirm?

Reply:
Thanks for the feedback :pray: 

> Should I open a PR in VTK (current VTK master)?

It would be great if you could open a MR against upstream VTK, then we would backport the change into our Slicer/VTK fork. Grazie :smile:

Reply:
MR done, see https://gitlab.kitware.com/vtk/vtk/-/merge_requests/7193

Reply:
Thanks David, waiting this is integrated in upstream project I backported the commit to our fork.

Reply:
perfect! :+1: